.github/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+<!-- If you're making a doc PR or something tiny where the below is irrelevant, delete this
+template and use a short description, but in your description aim to include both what the
+change is, and why it is being made, with enough context for anyone to understand. -->
+
+<details>
+  <summary>PR Checklist</summary>
+  
+### PR Structure
+
+* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
+* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
+  otherwise).
+* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
+  `services/friendbot`, or `all` or `doc` if the changes are broad or impact many
+  packages.
```

Comment:
This should be updated to be the same as https://github.com/stellar/stellar-dbt/pull/174/files#diff-b2496e80299b8c3150b1944450bd81c622e04e13d15c411d291db0927d75fd6bR13